### DISCUSSION TITLE
Error Connecting to Google Sheets

### DISCUSSION THREAD

**Discussion by alanfallow** [Budibase/budibase#9971] [Help] [answered]
Hi folks I keep getting this.  Where exactly does the ID go.
Thanks.

Error updating datasource schema : You must set a spreadsheet ID in your configuration to fetch tables.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @alanfallow,

Is it possible that you may have not saved before trying to fetch tables?
I've just come across the same issue when trying to recreate this.

<img width="968" alt="Error" src="https://user-images.githubusercontent.com/126772285/224269340-fe839806-8b9f-4fb5-81c4-ca91a60de685.png">

After saving and then running fetch tables it worked.

Didn't really have a lot of information to go on here.
I hope this helps.

**Reply by alanfallow** [reply]
Brilliant thanks
